spec/models/concerns/arclight/search_behavior_spec.rb

```diff
@@ -30,4 +30,32 @@
       end
     end
   end
+  describe '#add_hierarchy_sort' do
+    context 'when in hierarchy view' do
+      let(:user_params) { { view: 'hierarchy' } }
+
+      it 'adds component-order sort to query' do
+        expect(search_builder_instance.add_hierarchy_sort(solr_params)).to include(sort: 'sort_ii asc')
+      end
+    end
+    context 'when not in hierarchy view' do
+      it 'does not affect sort param' do
+        expect(search_builder_instance.add_hierarchy_sort(solr_params)).to eq({})
+      end
+    end
+  end
+  describe '#add_highlighting' do
```

Comment:
just reviewed this one with @jkeck. @drh-stanford, could you separate these highlighting tests into a different commit?